examples/apns.rb

```diff
@@ -13,6 +13,9 @@
 notification = RubyPushNotifications::APNS::APNSNotification.new tokens, { aps: { alert: 'Hello APNS World!', sound: 'true', badge: 1 } }
 
 pusher = RubyPushNotifications::APNS::APNSPusher.new(File.read('/path/to/your/apps/certificate.pem'), true, password) # enter the password if present
+# Connect timeout defaults to 30s
+# pusher = RubyPushNotifications::APNS::APNSPusher.new(File.read('/path/to/your/apps/certificate.pem'), true, password, connect_timeout: { 20 })
```

Comment:
`connect_timeout: { 20 }` doesn't make much sense, did you meant `{ connect_timeout: 20 }`? :)